app/models/server/models/Settings.js

```diff
@@ -242,6 +242,13 @@ export class Settings extends Base {
 			}
 		}
 	}
+
+	isOAuthEnabled() {
+		return !!this.findOne({
+			_id: { $regex: /^Accounts_OAuth_/ },
```

Comment:
This will return true for any OAuth related settings, not just the "enabled" settings. For example, `Accounts_OAuth_Gitlab_merge_users` is boolean and may be turned on even if the gitlab OAuth is off.

Reply:
updated the rex to only match against the servicce enabled documents